### DISCUSSION TITLE
Lightbar not working, only flashing taillights.

### DISCUSSION THREAD

**Discussion by Kraslot2023** [matsn0w/IIS-EmergencyLS-ELS-FiveM#145] [Q&A]
So I've finnally made the resource work for my server but whenever I spawn in a ELS vehicle and press "Q" it's just the taillights flashing. I've put my .xml fils inside folder 'xmlFiles'. Tried everything I could possibly know, but no succes.

Hope to find an answer here!

**Comment by matsn0w** [comment]
Hi, can you provide the VCF (.xml) you are using? Make sure you are using the very different MISS ELS syntax. See the [docs](https://github.com/matsn0w/MISS-ELS/blob/main/docs/vcf.md).
